demisto_sdk/__main__.py

```diff
@@ -558,31 +562,57 @@ def lint(**kwargs):
     "-y", "--assume-yes",
     help="Automatic yes to prompts; assume 'yes' as answer to all prompts and run non-interactively",
     is_flag=True)
-def format_yml(**kwargs):
-    return format_manager(**kwargs)
+def format(
+        input: Path,
+        output: Path,
+        from_version: str,
+        no_validate: bool,
+        update_docker: bool,
+        verbose: bool,
+        assume_yes: bool
+):
+    """Run formatter on a given script/playbook/integration/incidentfield/indicatorfield/
+    incidenttype/indicatortype/layout/dashboard/classifier/mapper/widget/report file.
+    """
+    return format_manager(
```

Comment:
seems I forgot to add the `check_configuration_file('format', kwargs)` here, we might want to keep this with kwargs for .conf file